### DISCUSSION TITLE
Users in docker group

### DISCUSSION THREAD

**Discussion by MC-Escherichia** [nix-community/NixOS-WSL#142] [Q&A] [answered]
Should adding the wsl user to the docker group with `wsl.docker-native.addToDockerGroup = true` allow for using docker without sudo?  I can use docker but it requires sudo, I'm wondering if this is the expected behavior.

**Comment by SuperSandro2000** [comment, marked as answer]
Yes, this is expected because this basically allows root escalation without user interaction.
